tests/coreml/test_cml_ImputerConverter.py

```diff
@@ -1,20 +1,31 @@
 """
 Tests CoreML Imputer converter.
 """
-import coremltools
 import numpy as np
 import unittest
-from sklearn.preprocessing import Imputer
+try:
+    from sklearn.impute import SimpleImputer as Imputer
+except ImportError:
+    from sklearn.preprocessing import Imputer
+import sklearn.preprocessing
 from onnxmltools.convert.coreml.convert import convert
 from onnxmltools.utils import dump_data_and_model
 
 
 class TestCoreMLImputerConverter(unittest.TestCase):
 
     def test_imputer(self):
-        model = Imputer(missing_values='NaN', strategy='mean', axis=0)
+        if not hasattr(sklearn.preprocessing, 'Imputer'):
+            # coremltools 3.1 does not work with scikit-learn 0.22
+            setattr(sklearn.preprocessing, 'Imputer', Imputer)
+        
+        try:
+            model = Imputer(missing_values='NaN', strategy='mean', axis=0)
+        except TypeError:
+            model = Imputer(missing_values='NaN', strategy='mean')
         data = [[1, 2], [np.nan, 3], [7, 6]]
         model.fit(data)
+        import coremltools  # noqa
```

Comment:
Why not put import coremltools after the sklearn import above?

Reply:
Just investigating.

Reply:
import coremltools fails if install with scikit-learn 0.22 so I moved it further. The test does not change but it fails later giving me more insights.